app/src/main/java/com/aritra/notify/ui/screens/notes/addEditScreen/AddEditScreen.kt

```diff
@@ -281,41 +285,56 @@ fun AddEditScreen(
                     .verticalScroll(rememberScrollState())
             ) {
                 if (isNew) {
-                    photoUri?.let {
-                        Log.d("URI", "Photo uri $photoUri")
-                        Row(
-                            modifier = Modifier.fillMaxWidth(),
-                            horizontalArrangement = Arrangement.End
+                    if (photoUri.isNotEmpty()) {
+                        LazyRow(
+                            modifier = Modifier.height(240.dp),
+                            horizontalArrangement = Arrangement.spacedBy(10.dp)
                         ) {
-                            IconButton(
-                                onClick = {
-                                    photoUri = null
+                            items(photoUri.size) {
+                                Box(
+                                    Modifier
+                                        .fillMaxHeight()
+                                        .width((LocalConfiguration.current.screenWidthDp * 0.8).dp)
+                                ) {
+                                    ZoomableAsyncImage(
```

Comment:
Make the image size small.
Try this code in the modifier of  `ZoomAsyncImage` & play with the height and width and see which looks better. On doing `fillMaxSize()` it's not looking good from my pov.
```
modifier = Modifier
           .width(120.dp)
           .height(150.dp)
           .padding(5.dp)
           l̥.clip(RoundedCornerShape(5.dp))
```

Reply:
The modifier size attributes were used in the box because the box holds both the image and icon button over it.  However I've tried to make it look better taking your inputs. Have a look :

<image src="https://github.com/aritra-tech/Notify/assets/124783808/994163f9-6ac5-4958-b8fb-7b32a508df07" height="800"/>

Reply:
Yeah looking better.
Nice work.
Can you also send a screenshot when the layout is changed to grid view @beradeep?

Reply:
Here's the layout in grid view: 😃 
In the video the borders of the `grid cards` are not properly visible due to poor video quality.

https://github.com/aritra-tech/Notify/assets/124783808/398ed84d-0abd-44e0-bba9-9855c34ca93c